renderer/models/dashboard/IJavaInitializer.ts

```diff
@@ -2,7 +2,7 @@ import { FormType } from './FormType';
 
 export interface IJavaInitializerForm {
   workspaceDir: string[];
-  formControls: FormControls;
+  formControls: FormControls | any;
```

Comment:
Be careful with any. Doing something | any is pointless because any is more general than something
So (something | any) <-- are the same --> any

Reply:
I have added because of some errors you showed while building an application. And let me the exact reason for the build issue.

![image](https://user-images.githubusercontent.com/29056094/91689211-3de5cb80-eb81-11ea-9302-5d6d6e40af52.png)

Reply:
Mmm okay